src/Desktop/UIAutomation/A11yAutomation.cs

```diff
@@ -40,20 +41,33 @@ private static IUIAutomation GetUIAutomationInterface()
         /// <param name="pid"></param>
         /// <returns>return null if fail to get an element by process Id</returns>
         public static DesktopElement ElementFromProcessId(int pid)
+        {
+            var elements = ElementsFromProcessId(pid);
+            return elements != null ? elements.First() : null;
+        }
+
+        /// <summary>
+        /// Get DesktopElements based on Process Id.
+        /// </summary>
+        /// <param name="pid"></param>
+        /// <returns>return null if fail to get elements by process Id</returns>
```

Comment:
This feels like it's missing a word. Maybe:
```suggestion
        /// <returns>return null if we fail to get elements by process Id</returns>
```